step-definitions/core/vue.js

```diff
@@ -35,20 +32,64 @@ Given(/^I log in to evan.network using vue( with )?(\w+)?$/, async (customPart,
   }, [], function(result) {
     this.assert.ok(result.value);
   });
+})
+
+Given(/^I log in to evan.network using vue( with )?(\w+)?$/, async (customPart, accountName) => {
+  const evan = setupEvan(client);
+
+  await client.url(`${ evan.baseUrl }#/dashboard.vue.evan`);
+  await client.execute(function() {
+    window.localStorage.setItem('evan-vault', '');
+    window.localStorage.setItem('evan-test-mode', true);
+    window.localStorage.setItem('evan-warnings-disabled', '{"payment-channel":true}');
+    window.localStorage.setItem('evan-language', 'en');
+    window.localStorage.setItem('evan-test-recaptchaId', '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI');
+    return true;
+  })
+  await client.pause(3000);
+  if (customPart && !evan.accounts[accountName]) {
+    throw new Error(`no account data found for account ${accountName}`);
+  }
+  const user = evan.accounts[accountName || 'default'];
 
   // vue, to define
-  await client.click(selectors.vueLogin1);
-  await client.waitForElementVisible(selectors.vueFreeInput, 10 * 1000);
-  await client.click(selectors.vueFreeInput);
-  await client.waitForElementVisible(selectors.vueRecoveryKey);
-  await client.setValue(selectors.vueRecoveryKey, [user.mnemonic]);
-  await client.waitForElementVisible(selectors.vueLogin2);
-  await client.click(selectors.vueLogin2);
-  await client.waitForElementVisible(selectors.vuePassword);
-  await client.setValue(selectors.vuePassword, [user.password]);
-  await client.pause(1000);
-  await client.click(selectors.vueUnlock);
+  // click on the "sign in" link
+  await client.waitForElementPresent(`a[href*="sign-in"]`, 60 * 1000);
+  await client.click(`a[href*="sign-in"]`)
+  await client.pause(2000);
```

Comment:
Not tested, but should work: 

```suggestion
await client.waitForElementPresent(`#mnemonicInput0`, 5 * 1000);
```